app/Observers/UserObserver.php

```diff
@@ -22,8 +25,19 @@ public function created(User $user)
             ->where('name', '=' , 'welcome-message')
             ->firstOrFail();
 
-        if (!$notificationTemplate->disabled) {
-            $user->notify($notificationTemplate->getDynamicNotification(compact('user')));
+        if (!$notificationTemplate->disabled && app(MailSettings::class)->mail_password !== null) {
+            try {
+                $user->notify($notificationTemplate->getDynamicNotification(compact('user')));
+            } catch (\Exception $e) {
+                // log the error to laravel logs
+                Log::error($e->getMessage());
+
+                if(app()->runningInConsole()) {
+                    $out = new ConsoleOutput();
```

Comment:
There is no console output when this is triggered by a job or user action. If this, for some reason, fails, it should never fail. It should just error out, which adds logs to the log files

Reply:
i would rather see the error catching for console output in the command instead. not add all kinds of error handeling in the method that is not ment to handle this. but in my opion you can leave the handeling out like mentioned above.

Reply:
It was also used for debug purposes, because when the email fails it threw the error and basically said goodbye and good luck to the user, thought this would make it clear for e-mail failures when used from the console: 
![image](https://user-images.githubusercontent.com/34610614/182229832-3d7fa849-98a6-45e0-91a7-e45855f232f1.png)